src/components/LandDetailPage/UnsetENSContentModal.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import { T, t } from 'decentraland-dapps/dist/modules/translation/utils'
+import { Button, Modal } from 'decentraland-ui'
+import { ENS } from 'modules/ens/types'
+import { Land } from 'modules/land/types'
+import { Link } from 'react-router-dom'
+import { locations } from 'routing/locations'
+
+type UnsetENSContentModalProps = {
+  land: Land
```

Comment:
I'll leave this to @cazala but maybe it's a good idea to connect this component to te state instead of passing down the [prop here](https://github.com/decentraland/builder/pull/1534/files#diff-bd96a13dbca6fcb2f285a65246189d82dd9cff7d5ff348090283b64f7dcaa64aR8)

I say this because ENSChip needing a Land prop to work is a bit weird

Reply:
Right, It is now connected to the store and decoupled from the chip, land is no longer provided